src/components/SignUp/SignUp.test.tsx

```diff
@@ -59,7 +60,7 @@ describe('SignUp component', () => {
 
     it('should render captcha block', () => {
         const wrapper = setup({hasConfirmed: true, captchaType: 'recaptcha'});
-        expect(wrapper.find('.cr-sign-up-form__recaptcha').exists()).toBe(true);
+        expect(wrapper.find('.cr-sign-up-form__recaptcha').exists()).toBe(false);
```

Comment:
if type is 'recaptcha' block '.cr-sign-up-form__recaptcha' are checking to not exist?

Reply:
because we have ```captchaType: 'none',```